R/compute_duration.R

```diff
@@ -63,7 +63,8 @@
 #'
 #' @details The duration is derived as time from start to end date in the
 #'   specified output unit. If the end date is before the start date, the
-#'   duration is negative.
+#'   duration is negative. Usually this computation function can not be used
+#'   with %>%.
```

Comment:
Please use backticks for `%>%`. Then it is formatted as code.

Reply:
Updated. Thanks!